xml/System.Reflection.Metadata/MetadataReader.xml

```diff
@@ -46,7 +46,7 @@
       <Docs>
         <param name="metadata">To be added.</param>
         <param name="length">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Initializes a new instance of the <see cref="T:System.Reflection.Metadata.MetadataReader" /> class from the metadata stored at the given memory location.</summary>
```

Comment:
The descriptions here all look the same: https://review.docs.microsoft.com/en-us/dotnet/api/system.reflection.metadata.metadatareader.-ctor?branch=pr-en-us-2333&view=netcore-2.2

We can merge this as-is but it's not a good experience. Also, because the parameters are not filled out, the number of undocumented APIs won't drop here.

Reply:
@tmat @nguerrera it would be great if you can help us define better descriptions. I'll submit the changes we found from the triple slash comments, but they should be improved in future iterations.

Reply:
@mairaw Can we merge this PR as is, or should we wait for the area owners to give better suggestions?

Reply:
Let's merge as-is. These APIs will still need to be reviewed since params are also not filled out.